mm2src/coins/eth.rs

```diff
@@ -5593,7 +5596,15 @@ pub async fn eth_coin_from_conf_and_request(
         erc20_tokens_infos: Default::default(),
         abortable_system,
     };
-    Ok(EthCoin(Arc::new(coin)))
+
+    let coin = EthCoin(Arc::new(coin));
+    if let Some(stream_config) = &ctx.event_stream_configuration {
+        if let EventInitStatus::Failed(err) = EventBehaviour::spawn_if_active(coin.clone(), stream_config).await {
+            return ERR!("Failed spawning balance events. Error: {}", err);
+        }
+    }
+
```

Comment:
what if we move this to `EthCoin` by method and call `coin.spawn_balance(&ctx)` instead of repeating this code block here and in `eth_coin_from_conf_and_request_v2`

this can be better when/if this code block needs to be modified in the future then it will only be done in one place 

```rust
impl EthCoin {
    async fn spawn_balance(&self, ctx: &MmArc) -> MmResult<(), EthActivationV2Error> {
        if let Some(stream_config) = &ctx.event_stream_configuration {
            if let EventInitStatus::Failed(err) = EventBehaviour::spawn_if_active(self.clone(), stream_config).await {
                return MmError::err(EthActivationV2Error::FailedSpawningBalanceEvents(err));
            }
        }

        Ok(())
    }
}
```

 creating a new trait for `BalanceSpawn` can also work in case it's need for other chains e.g
```rust
#[async_trait]
trait BalanceSpawnOps {
    type Error;
    async fn spawn_balance(&self, ctx: &MmArc) -> Result<(), Self::Error>;
}

#[async_trait]
impl BalanceSpawnOps for EthCoin {
    type Error = EthActivationV2Error;
    async fn spawn_balance(&self, ctx: &MmArc) -> Result<(), Self::Error> {
        if let Some(stream_config) = &ctx.event_stream_configuration {
            if let EventInitStatus::Failed(err) = EventBehaviour::spawn_if_active(self.clone(), stream_config).await {
                return Err(EthActivationV2Error::FailedSpawningBalanceEvents(err));
            }
        }
        Ok(())
    }
}
```